inference-engine/thirdparty/clDNN/runtime/kernels_cache.cpp

```diff
@@ -6,6 +6,7 @@
 #include "kernels_cache.hpp"
 #include "ocl/ocl_engine.hpp"
 #include "cldnn/runtime/debug_configuration.hpp"
+#include "kernel_selector/core/common/primitive_db.h"
```

Comment:
Can we avoid dependency of runtime lib from kernel selector?

Reply:
I think we can make a new class independent from primitive_db. 
I just used primitive_db to reuse the class, but if you think it is better to have separate one, I will proceed with it.

Reply:
Maybe this batch headers code can be added into `kernels_code`?

Reply:
Ah that can be better. Thanks!

Reply:
Hmm kernels_code is a set of kernel_code so cannot get the target string quickly and also it has redundant info, while we just need a string. 
I am thinking of include the batch_headers.inc file  here directly and define it as a global static string. 
(I think its better to use it as global because using too long stack allocated string is not good) 
Let me apply my suggestion soon.

Reply:
Updated as I mentioned. Please take a look!

Reply:
This is not work due to the dependencies to the codegen codes, which is not being recognized pretty good. Maybe I need to major CMakeFile to have the codegen directory path. However that is not good looking. So, I will consider another flow which can pass the strigns from kernel selector through program to runtime..

Reply:
I changed the flow, to pass the str from program to kernels_cache. Could you review again?